ynput/OpenPype
# Maya: Attributes are locked after publishing if they are locked in Camera Family

## Changelog Description
This PR is to make sure unlock attributes only during the bake context, make sure attributes are relocked after to preserve the lock state of the original node being baked.
## Additional info
It only happens when the camera is exported in raw maya scene(ma format).

## Testing notes:
1. Launch Maya via Launcher
2. Create Camera Instance
3. Freeze some Camera Attributes
4. Publish
5. Load the published camera by reference ma.
6. The camera attributes should be locked if the camera attributes not being frozen before publishing
